tests/syrupy/extensions/amber/__snapshots__/test_amber_matchers.ambr

```diff
@@ -1,3 +1,4 @@
+# serializer version: 1
```

Comment:
In theory, this may let us support multiple concurrent versions of a serializer without requiring a major release.